src/ray/core_worker/test/mock_worker.cc

```diff
@@ -128,6 +130,14 @@ class MockWorker {
     return Status::OK();
   }
 
+  Status WhileTrueLoop(const std::vector<std::shared_ptr<RayObject>> &args,
+                       const std::vector<ObjectID> &return_ids,
+                       std::vector<std::shared_ptr<RayObject>> *results) {
+    while (1) {
+    }
```

Comment:
How about add a sleep to reduce CPU load?

Reply:
that's a good idea! I should add that